jetty-server/src/test/java/org/eclipse/jetty/server/session/FileSessionManagerTest.java

```diff
@@ -195,4 +196,52 @@ public boolean accept(File dir, String name)
         
         server.stop();
     }
+
```

Comment:
I copied much of this test method from `testDangerousSessionIdRemoval`. It fails without my changes above (with the `StringIndexOutOfBoundsException` mentioned in #1373).